packages/nextjs/app/builders/[0x78F348Dc5De2e9f066FAAa0bfD66d397a9260A43]/page.tsx

```diff
@@ -0,0 +1,80 @@
+"use client";
```

Comment:
We don't need this restriction since your page doesn't use any client-only mechanism.

Please remove this line so we can improve the app's performance by pre-rendering the page during the compile process.